### PR TITLE
Remove comments, move DOM creation to html

### PR DESCRIPTION
**Pull request by SamBroner**
I'm mixing 2 concerns with this PR, so feel free to agree with only one part.

1. Remove comments... it was just getting hard to read. Ideally we should be more self explanatory.

2. Move DOM creation to HTML... Shortened fn significantly and improved readability. On the flip, encapsulation into code proposal may be harder with this model, although it could be done through webpack.

### CODE REVIEW

**Comment by micahgodbolt** [comment]
yeah, i wasn't a huge fan of the notes overload. happy to see them go. love the idea of moving DOM into the HTML file since that's not really part of the exercise. We should do that for all dice demos

File: examples/hosts/app-integration/external-controller/src/view.ts [hunk 1]
```diff
@@ -6,29 +7,21 @@
...
     const dataKey = "dataKey";
     // Set init value for dataKey
     DO.set(dataKey, 1);
-    const wrapperDiv = document.createElement("div");
-    wrapperDiv.style.textAlign = "center";
-    div.append(wrapperDiv);
 
-    const diceCharDiv = document.createElement("div");
-    diceCharDiv.style.fontSize = "200px";
+    const diceCharDiv = document.getElementById("dice")!;
```
Comment (ChumpChief) [inline examples/hosts/app-integration/external-controller/src/view.ts:19]: I'm actually not a fan of this :-/  It imposes an external dependency on the consumer of the view, basically reducing the encapsulation of the view.  It also prevents multiply-rendering the view (e.g. today you can render the dice twice, or render two different dice rollers by just calling this same render function twice).

**Comment by SamBroner** [comment]
> yeah, i wasn't a huge fan of the notes overload. happy to see them go. love the idea of moving DOM into the HTML file since that's not really part of the exercise. We should do that for all dice demos

I'm not sure if we should remove it for everything. There's a question on how much of the DOM Fluid should encapsulate. Curious what @ChumpChief thinks...

File: examples/hosts/app-integration/external-controller/src/view.ts (hunk 1 above)
Comment (SamBroner) [reply examples/hosts/app-integration/external-controller/src/view.ts:19]: Let's have the dialogue here... It's a good question.

Comment (SamBroner) [reply examples/hosts/app-integration/external-controller/src/view.ts:19]: I think if we're in the data + API layer, then we'll mostly have developers not encapsulate the view. Is that something we're *un*comfortable with?

Comment (ChumpChief) [reply examples/hosts/app-integration/external-controller/src/view.ts:19]: A view author using a view framework (React, Vue, etc.) will have encapsulated view components.  And even for vanillaJS views, if they are consuming our data + API layer to build a reusable component (e.g. drawing multiple poll widgets on a single page) then encapsulation and multiple-rendering is likely still valuable to them.  I think the view pattern is independent from their Fluid consumption pattern.

Comment (micahgodbolt) [reply examples/hosts/app-integration/external-controller/src/view.ts:19]: true...it's not really a view anymore, it'd be a controller

Comment (SamBroner) [reply examples/hosts/app-integration/external-controller/src/view.ts:19]: Yep. We should pick an agenda for this directory. If it's "Hello World", then I think moving DOM creation to the html is reasonable. That change simplifies things and isn't meant to be indicative of the whole app structure.

If this is an exploration of the surface, then we keep the vie controller coupled with the view to mimic the more common view library patterns.

Thoughts?

**Comment by SamBroner** [comment]
I shouldn't have mixed concerns. I'll close this and open another.

**Pull request closed without merging**